Fiber2D/ActionRepeat.swift

```diff
@@ -37,14 +37,12 @@ public struct ActionRepeatForeverContainer: ActionContainer {
                 }
             }
             
-            if let target = target {
-                innerContainer.start(with: target)
-            }
+            innerContainer.start(with: target)
         }
     }
     
     public var tag: Int = 0
-    weak var target: Node?
+    weak var target: Node!
```

Comment:
This is actually meaningless. Replace all these with `unowned` and we are ready to merge